src/dal/lucky.ts

```diff
@@ -0,0 +1,29 @@
+import { database } from "../libs/database";
+import { getRandomNumberBetweenTwoNumbers } from "../randomizer";
+import { numbersSignsBG_zodiacDirBg } from "../models/signs";
+
+export async function getLuckyTextByRandomNumber(): Promise<string> {
```

Comment:
Here the naming lies, you're not getting lucky text by random number, you're not passing any random number, so this function might have to be called **getLuckyText**?